silverback/_cli.py

```diff
@@ -304,41 +304,40 @@ def new_workspace(
 @click.option(
     "-n",
     "--name",
-    "update_name",
+    "name",
     help="Update name for workspace",
```

Comment:
Make sure to add this
```suggestion
    defualt=None,
    help="Update name for workspace",
```